test/mocha/10-api.js

```diff
@@ -71,6 +72,38 @@ describe('bedrock-profile-http', () => {
       result.ok.should.equal(true);
       result.data.id.should.be.a('string');
     });
+    it('create a new profile with didMethods "v1" and "key"', async () => {
```

Comment:
you don't need to redo this, but this does work:

```js
const didMethods = ['v1', 'key'];

for(const didMethod of didMethods) {
  it(`create a new profile with didMethod ${didMethod}`, async function() {

  })
}
```

this would result in the test suite having 2 tests one for each method which would clarify which methods are failing on failure.